packages/react-form/src/hooks/field/reducer.ts

```diff
@@ -48,6 +48,30 @@ export function updateErrorAction(error: ErrorValue): UpdateErrorAction {
   };
 }
 
+function shallowArrayComparison(arrA, arrB) {
```

Comment:
nit: could we put this into [utilities](https://github.com/Shopify/quilt/blob/master/packages/react-form/src/utilities.ts)

Reply:
Also, we should type them as `unknown[]`

Reply:
Sure thing. I've moved it there.